desktop/core/src/desktop/js/reactComponents/FileChooser/PathBrowser/PathBrowser.tsx

```diff
@@ -97,125 +92,125 @@ const PathBrowser = ({
   };
 
   if (breadcrumbs) {
+    if (isEditMode) {
+      return (
+        <div ref={wrapperRef}>
+          <Input
+            prefix={showIcon ? icons[fileSystem] : <span />}
+            defaultValue={decodeURIComponent(breadcrumbs[breadcrumbs.length - 1].url)}
```

Comment:
Where are the urls of the breadcrumbs coming from? Does it make sense for them to be url encoded here already?

Reply:
Make sense,
I checked, in new public apis, all the path are not encoded, hence I have removed the decoding logic.
Thanks for highlight.